python/ray/data/_internal/logging_json.yaml

```diff
@@ -0,0 +1,34 @@
+version: 1
```

Comment:
what is this yaml file format? Is this somethign specific to ray data? is there documentation about this?

Reply:
The YAML file format is specific to python (more docs [here](https://docs.python.org/3/library/logging.config.html)). We just directly load the yaml with something like
```python
    with open(config_path) as file:
        config = yaml.safe_load(file)
    logging.config.dictConfig(config)
```

Probably could do a better job explaining that this could be used in the brief logging docs [here](https://docs.ray.io/en/latest/data/monitoring-your-workload.html#ray-data-logs).

Reply:
maybe add a comment in this file explaining the origin/format of this YAML file?